tests/switchless/enc/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+oeedl_file(../switchless.edl enclave gen)
```

Comment:
Please don't use `oeedl_file` in new code, you'll run into dependency issues. See the deep-copy tests for up-to-date examples.

Reply:
Good suggestion. Will do.